### DISCUSSION TITLE
Hacktober - Will FreshRSS take part?

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3838] [Ideas]
Have you heard from https://hacktoberfest.digitalocean.com/ ?

I think it could be a cool idea to take part as FreshRSS. What do you think?

**Comment by Alkarex** [comment]
I am not familiar with it, but sounds good :-)
I have only quickly looked at the description, and I am bit unsure of what can be prepared in advance (e.g. mark a repository as participating?), and what the rules are more concretely. Summary welcome :-)

**Reply by math-GH** [reply]
As I understood we need just to label issues with "Hacktoberfest" (see all labeled issues: https://github.com/search?q=label%3Ahacktoberfest+state%3Aopen&type=Issues) and than the participant will find it and will send PRs that we have to check.

see: https://hacktoberfest.digitalocean.com/resources/maintainers

**Reply by math-GH** [reply]
I would suggest to create an i18n issue that we need to update our i18n gaps and label it with "hacktoberfest"

**Reply by Frenzie** [reply]
Which gaps?

**Reply by math-GH** [reply]
I mean the "Todos" in the i18n files.

**Reply by math-GH** [reply]
f.e. the he (hebraic)
![grafik](https://user-images.githubusercontent.com/1645099/132981821-126a7c19-2442-4490-acb7-c11ce7200ad7.png)

**Reply by Frenzie** [reply]
Gaps is a dramatic word for a few new strings, but sure, why not.

**Reply by Alkarex** [reply]
I have created the label https://github.com/FreshRSS/FreshRSS/labels/Hacktoberfest
